cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,62 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
-describe('Settings page', () => {
-  before(() => {
-
-  });
+const faker = require("faker");
```

Comment:
Cypress is bundled with Mocha and Chai, but it does not play well with modules that use global variables, like require. Instead use ES6 import statement like import faker from 'faker'.